etl/__main__.py

```diff
@@ -283,7 +300,7 @@ def main():
                             if "extra_keys" in metric.definition
                             else None,
                             type=metric.definition["type"],
-                            expires=metric.definition["expires"],
+                            expires=get_expiry_date(metric.definition["expires"], app_name),
```

Comment:
I think we also want to calculate an explicit `expiry_date` property which we can use to determine when something is expired. Probably worth writing a seperate method (distinct from `get_expiry_text`) to do just that.